pkg/agent/agent.go

```diff
@@ -291,7 +291,7 @@ func (i *Initializer) Initialize() error {
 	wg.Add(1)
 	// routeClient.Initialize() should be after i.setupOVSBridge() which
 	// creates the host gateway interface.
-	if err := i.routeClient.Initialize(i.nodeConfig, wg.Done); err != nil {
+	if err := i.routeClient.Initialize(i.nodeConfig, i.enableProxy, wg.Done); err != nil {
```

Comment:
why passing i.enableProxy? I think the feature should be managed by proxyAll as well, which is already a member variable of routeClient?

Reply:
This PR has not been rebased on Linux one. Now it is updated.